SConstruct

```diff
@@ -1,5 +1,7 @@
-# panda fw
-SConscript('board/SConscript')
+AddOption('--test',
+          action='store_true',
+          default=True,
+          help='build test files')
```

Comment:
Verified that openpilot's option overrides panda's here when running `scons` from the root directory, so test still won't be built on device.